stock_dock/readme/CONTRIBUTORS.rst

```diff
@@ -0,0 +1,9 @@
+* Sébastien Alix <sebastien.alix@camptocamp.com>
+
+Design
+~~~~~~
+
+* Joël Grand-Guillaume <joel.grandguillaume@camptocamp.com>
+* Jacques-Etienne Baudoux <je@bcim.be>
+* `Trobz <https://trobz.com>`_:
```

Comment:
this should be moved above, under  Sébastien name

Reply:
@simahawk thx, I updated the PR